extensions/interactions/TextInput/TextInput.html

```diff
@@ -9,13 +24,14 @@
   <div class="row text-input-container">
     <form ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
       <div class="col-lg-12 col-md-12 col-sm-12">
-        <schema-based-editor schema="schema" local-value="answer" label-for-focus-target="labelForFocusTarget">
-        </schema-based-editor>
-      </div>
-      <div ng-show="rows > 1">
-        <button type="submit" class="btn btn-default" style="margin-top: 15px; margin-left: 16px;">
-          Submit
-        </button>
+        <input ng-hide="rows > 1" type="text" ng-model="answer" class="text-input-single-line-field form-control" placeholder="{{placeholder}}" focus-on="<[labelForFocusTarget]>">
+          <div ng-show="rows > 1">
```

Comment:
Hang on, I think you over-indented. This div should match exactly the level of the indentation of the previous 'input' tag, since both tags are siblings and on the same 'level'.